node/src/cli.rs

```diff
@@ -38,4 +38,8 @@ pub enum Subcommand {
     /// The custom benchmark subcommmand benchmarking runtime pallets.
     #[structopt(name = "benchmark", about = "Benchmark runtime pallets.")]
     Benchmark(frame_benchmarking_cli::BenchmarkCmd),
+
+    /// Supra CLI
```

Comment:
Can we give it more descriptive comment like: "Decodes a base58 PeerID and returns its hex and bytes forms"

Reply:
Done